docs/en/conf.md

```diff
@@ -0,0 +1,189 @@
+# Configuration file description
+
+## nameserver configuration file conf/nameserver.flags
+
+```
+# nameserver.conf
+--endpoint=127.0.0.1:6527
+--role=nameserver
+# If you are deploying the standalone version, you do not need to configure zk_cluster and zk_root_path, just comment these two configurations. Deploying the cluster version needs to configure these two items, and the two configurations of all nodes in a cluster must be consistent
+#--zk_cluster=127.0.0.1:7181
+#--zk_root_path=/openmldb_cluster
+# The address of the tablet needs to be specified in the stand-alone version, and this configuration can be ignored in the cluster version
+--tablet=127.0.0.1:9921
+# Configure log directory
+--openmldb_log_dir=./logs
+# Configure whether to enable automatic recovery. If it is enabled, the node will automatically perform the leader switch if it hangs, and the data will be automatically restored after the node process starts.
+--auto_failover=true
+
+# Configure the thread pool size, no need to modify
+#--thread_pool_size=16
+# Configure the number of failed retries, the default is 3
+#--request_max_retry=3
+# Configure the request timeout, the default is 12 seconds
+#--request_timeout_ms=12000
+# Configure the retry interval when the request is unreachable, generally do not need to be modified
+#--request_sleep_time=1000
+# Configure the zookeeper session timeout in milliseconds
+--zk_session_timeout=10000
+# Configure the zookeeper health check interval, the unit is milliseconds, generally do not need to be modified
+#--zk_keep_alive_check_interval=15000
+# Configure the timeout period for tablet heartbeat detection, the default is 1 minute. If the tablet is still connected after this time, the nameserver considers that the tablet is unavailable and will perform the operation of offline the node
+--tablet_heartbeat_timeout=60000
+# Configure the tablet health check interval, in milliseconds
+#--tablet_offline_check_interval=1000
+
+# The number of queues to perform high-availability tasks
+#--name_server_task_pool_size=8
+# The number of concurrent execution of high-availability tasks
+#--name_server_task_concurrency=2
+# The maximum number of concurrent execution of high-availability tasks
+#--name_server_task_max_concurrency=8
+# Check the waiting time of the task when executing the task
+#--name_server_task_wait_time=1000
+# The maximum time to execute the task, if it exceeds, it will log
+#--name_server_op_execute_timeout=7200000
+# Get the time interval of the task
```

Comment:
The time interval of receiving the status of the next task
(the Chinese doc is also a bit confusing)

Reply:
ok. Thank you.